CONTRIBUTING.md

```diff
@@ -89,14 +89,15 @@ run in as many environments as possible. To paraphrase the
 Consider that a well-crafted `EXPECTF` section may allow a `SKIPIF` section to
 be less restrictive.
 
-### Local Mongo Orchestration (and Travis CI)
+### Local Mongo Orchestration (and CI platforms)
 
 The test suite depends on [Mongo Orchestration](https://github.com/10gen/mongo-orchestration).
 Mongo Orchestration is an HTTP server that provides a REST API for maintaining
-MongoDB configurations. These configurations are located in ``scripts/presets``
-and for Travis CI in ``scripts/presets/travis``. The presets for Travis CI can
-also be spun-up locally, and that is the preferred testing method. An older way
-using a specific VM is also still available (see further down).
+MongoDB configurations. These configurations are provided by the
+[drivers-evergreen-tools](https://github.com/mongodb-labs/drivers-evergreen-tools)
+repository. These configurations can be run locally. Alternatively, you can use
+the GitHub Actions workflow if you don't want to set up a local development
+environment.
```

Comment:
This section may be a good place to mention that the `MONGO_ORCHESTRATION_URI` environment variable can be defined if the default `http://localhost:8889/v1` is not suitable. Perhaps you can add a line below the shell command that starts MO.

Reply:
Added 👍